client/package.json

```diff
@@ -18,7 +19,9 @@
     "@types/node": "^20",
     "@types/react": "^18",
     "@types/react-dom": "^18",
-    "eslint": "^8",
+    "@typescript-eslint/eslint-plugin": "^6.10.0",
+    "@typescript-eslint/parser": "^6.10.0",
+    "eslint": "^8.53.0",
     "eslint-config-next": "13.5.4",
```

Comment:
**_Review comment 2_**

The version of the eslint has also changed in the package.json file. Can we stick to the eslint package version that was there before? Or you changed it because a problem occured? @Akpjunior94

Reply:
@Ifycode in the process of installing the other two packages the eslint got updated automatically. I think the symbol ^ means it can install latest within version 8 right?

Reply:
Well I think I can run it in such a way it doesn't update the eslint. and going forward whenever I am adding new dependencies then it shouldn't update or upgrade affect other ones

Reply:
I just only want to see changes made in a pull request. Except if it gives problems and breaks code that's the only reason it should change in the same pull request @Akpjunior94 
If it doesn't break anything, but you feel it must or can change, then it belongs to another pull request. Not this one 👍🏼

Reply:
Things are easy to keep track of this way @Akpjunior94

Reply:
noted

Reply:
Eslint reverted to previous version.